guides/common/modules/proc_importing-kickstart-repositories.adoc

```diff
@@ -28,7 +28,7 @@ endif::[]
 ----
 
 ifeval::["{rhel-version}" == "8"]
-. Create directories for {RHEL}Linux 8 AppStream and BaseOS Kickstart repositories:
+. Create directories for {RHEL} 8 AppStream and BaseOS Kickstart repositories:
```

Comment:
```suggestion
. Create directories for {EL} 8 AppStream and BaseOS Kickstart repositories:
```

Reply:
Below it is the instruction: `# mkdir --parents /var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/appstream/kickstart` which also mentions rhel in the path name. I think `{RHEL}` is correct here.